### PR TITLE
cownomics patch

### PR DESCRIPTION
**Pull request by SansPapyrus683**
_If any of the below don't apply to this Pull Request, mark the checkbox as done._

- [x] I have tested my code.
- [x] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [x] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions), which include the following: 
  - I understand that if it is clear that I have not attempted to follow these guidelines (ex. if I have not used tabs to indent), my PR will be closed.
  - If changes are requested, I will re-request the review after making the changes.

### CODE REVIEW

File: solutions/usaco-736.mdx [hunk 1]
```diff
@@ -86,120 +147,182 @@ public class BovineGeonomics {
...
+	int poss_positions = 0;
+	for (int i = 0; i < m; i++) {
+		std::set<char> seen;
+		for (int j = 0; j < n; j++) {
+			seen.insert(plain_cows[j][i]);
+		}
+
+		bool dupe = false;
+		for (int j = 0; j < n; j++) {
+			if (seen.count(spotted_cows[j][i])) {
```
Comment (ChuangSheep) [inline solutions/usaco-736.mdx:235]:
```suggestion
			if (seen.count(spotted_cows[j][i]) > 0) {
```

(actually we can simplify this part

```
		bool dupe = false;
		for (int j = 0; j < n && !dupe; j++) {
			dupe = seen.count(spotted_cows[j][i]) > 0;
		}
```

**Review by ChuangSheep — APPROVED** [review]
nice

File: solutions/usaco-736.mdx (hunk 1 above)
Comment (SansPapyrus683) [reply solutions/usaco-736.mdx:235]: i don't think that'll work bc if there's a false at the end it won't register

**Comment by SansPapyrus683** [comment]
damit @ryanchou-dev review

**Comment by ryanchou-dev** [comment]
oops

**Review by ryanchou-dev — APPROVED** [review]

**Pull request merged**